script/run-gn-format.py

```diff
@@ -0,0 +1,23 @@
+import os
+import subprocess
+import sys
+
+SOURCE_ROOT = os.path.dirname(os.path.dirname(__file__))
+
+# Helper to run gn format on multiple files
+# (gn only formats a single file at a time)
+def main():
+  new_env = os.environ
```

Comment:
this doesn't actually copy the dictionary
```suggestion
  new_env = os.environ.copy()
```